src/test/resources/com/puppycrawl/tools/checkstyle/checks/imports/InputUnusedImportFromJavaLang.java

```diff
@@ -0,0 +1,44 @@
+package com.puppycrawl.tools.checkstyle.checks.imports;
+
+import java.lang.String; // VIOLATION
+import java.lang.Math; // VIOLATION
+import java.lang.Class; // VIOLATION
+import java.lang.Exception; // VIOLATION
+import java.lang.Runnable; // VIOLATION
+import java.lang.RuntimeException; // VIOLATION
+import java.lang.ProcessBuilder; // VIOLATION
+import java.lang.Double; // VIOLATION
+import java.lang.Integer; // VIOLATION
+import java.lang.Float; // VIOLATION
+import java.lang.Short; // VIOLATION
+
+import java.lang.annotation.Annotation; // OK
+import java.lang.reflect.Modifier; // OK
+import java.lang.reflect.Field; // OK
+
+import java.lang.*; // OK
```

Comment:
@MEZk Reason was most likely because of lack of context.
`some.package` may have 1 or multiple classes in it. If they just use class name alone, `MyClass`, we can't tell if it came from `some.package` or `java.lang` or another wildcard import. We would have to read the classpath to understand what classes are in the package.

In this instance, I don't feel we have to worry about the `java.lang` one because we are marking everything in just that package as redundant regardless of if it was used or not.
But @romani can decide otherwise.